debian/rules

```diff
@@ -7,6 +7,10 @@
 # sensible basis of options (eg no -O3, including -g). Debian specific options
 # come from CFLAGS as usual.
 
+override_dh_auto_clean:
```

Comment:
This new stanza ended up between two comment paragraphs that were intended to stay together, can you adjust it?

There is probably no reason to call ninja clean, rm -rf build-deb is all that is needed.